src/common/ipfixlolib/ipfixlolib.h

```diff
@@ -125,9 +125,16 @@ See ipfixlolib.h for details on how to use this library.
 extern "C" {
 #endif
 
+/* Supported export protocols */
+typedef enum export_protocol_version {
```

Comment:
I agree with @nickbroon. Let's get rid of the defines and use the enum instead :+1: